functional_tests/playbooks/create.yml

```diff
@@ -0,0 +1,61 @@
+# ansible-playbook functional_tests/playbooks/create.yml -e "file_name=my_file_name" -e "file_content='my file content'"
+
+---
+- hosts: localhost
+  connection: local
+  become: False
+  gather_facts: False
+  vars:
+    squest_token: 5a5554bdb3ad397721fe2a57e1a76a2df6279725
+    squest_api_url: http://127.0.0.1:8000/api
+    squest:
+      request:
+        instance:
+          id: 1
+          name: test
+          service: 1
+          spec: {}
+          state: PROVISIONING
```

Comment:
this sis used for testing. 
Need to be removed or it will take precedence on the squest call